docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,98 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+Whenever a masternode comes online or a client is syncing, they will send this message which describes the masternode entry and how to validate messages from it.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(signmnb)
```

Comment:
What does signmnb mean?  Is it signed?  If so, can you update the description to include the word signed?

If not why not just call the parameter "message"?